src/test/java/spoon/test/prettyprinter/SniperAnnotatedEnumTest.java

```diff
@@ -0,0 +1,59 @@
+package spoon.test.prettyprinter;
+
+import org.apache.commons.io.*;
+import org.hamcrest.*;
+import org.junit.*;
+import static org.junit.Assert.*;
+
+import spoon.Launcher;
+import spoon.compiler.*;
+import spoon.reflect.*;
+import spoon.reflect.code.CtComment.*;
+import spoon.reflect.declaration.*;
+import spoon.reflect.visitor.filter.*;
+import spoon.support.sniper.*;
+import spoon.test.GitHubIssue;
+
+import java.io.*;
+import java.nio.charset.*;
+import java.nio.file.*;
+
+public class SniperAnnotatedEnumTest {
+  private static final Path INPUT_PATH = Paths.get("src/test/java/");
+  private static final Path OUTPUT_PATH = Paths.get("target/test-output");
+
+  @BeforeClass
+  public static void setup() throws IOException {
+    FileUtils.deleteDirectory(OUTPUT_PATH.toFile());
+  }
+
+  @Test
```

Comment:
`GithubIssue` already serves as test annotation, so `@Test` can be removed.

Reply:
OK, I have removed the redundant test annotation